frontend/components/standard/Footer.vue

```diff
@@ -15,14 +16,36 @@ const tt = (s: string) => t(`${prefix}.${s}`)
       >Rocky Mountain Institute</a></span>
       <span class="text-600">© 2023 RMI</span>
     </div>
-    <div class="flex flex-row gap-3 align-items-center">
-      <PVButton
-        icon="pi pi-bolt"
-        class="p-1 w-auto"
-        :class="showStandardDebug ? '' : 'p-button-text'"
-        @click="() => showStandardDebug = !showStandardDebug"
-      />
+    <div class="flex flex-column align-items-center">
+      <div class="flex gap-1">
+        <PVButton
+          v-tooltip.top="tt('Manage Fake Users')"
+          icon="pi pi-user-plus"
+          class="p-1 w-auto"
+          :class="fakeUsersVisible ? '' : 'p-button-text'"
+          @click="() => fakeUsersVisible = !fakeUsersVisible"
+        />
```

Comment:
Should this be hidden behind some "only in dev" flag?

Reply:
^^

Reply:
Certainly at some point!